src/index.html

```diff
@@ -7,6 +7,196 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--sat calendar--28">
      <div class="calendar__cell">
      <div class="calendar__cell">
      <div class="calendar__cell">
      <div class="calendar__cell">
      <div class="calendar__cell">
      ....
    </div>
```
Вот так должна выглядеть структура твоего html, у каждой ячейки в календаре по одному классу, а у обертки всех ячеек с классом `calendar` должно быть еще два модифицированных класса `calendar--fri` и `calendar--29` которые отвечают за первый день месяца и количество дней в месяце соответственно